sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/SentryAutoConfiguration.java

```diff
@@ -198,9 +195,16 @@
           final @NotNull IHub hub) {
         FilterRegistrationBean<SentryTracingFilter> filter =
             new FilterRegistrationBean<>(new SentryTracingFilter(hub));
-        filter.setOrder(Ordered.HIGHEST_PRECEDENCE);
+        filter.setOrder(Ordered.HIGHEST_PRECEDENCE + 1); // must run after SentrySpringFilter
```

Comment:
nit: Could be worth then declaring these as field and name them so the fact one depends (must follow) on the other can never go out of sync:

```suggestion
        private static int SENTRY_SPRING_FILTER_PRESEDENCE = Ordered.HIGHEST_PRECEDENCE;
        // must run after SentrySpringFilter
        private static int SENTRY_TRACING_FILTER_PRESEDENCE + SENTRY_SPRING_FILTER_PRESEDENCE + 1;
        filter.setOrder(SENTRY_TRACING_FILTER_PRESEDENCE);
```

Reply:
👍